blue-xdma/Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: check_build_folder compile create sim synth clean
+
+.DEFAULT_GOAL := synth
+
+PCIE_LOC ?= "X0Y1"
+
+check_build_folder:
+	@if [ ! -d "build" ]; then \
+		mkdir build; \
+	fi
+
+compile: check_build_folder
+	$(MAKE) -C ./src
+
+create: compile
+	@if [ ! -f "./build/xdma_ex/xdma_ex.xpr" ]; then \
+		vivado -mode batch -notrace -nolog -nojournal -source ./script/create.tcl -tclargs $(PCIE_LOC); \
+    fi
```

Comment:
use TAB for indentation in Makefile

Reply:
The issue has already been resolved in the latest commit.